Dockerfile

```diff
@@ -1,15 +1,19 @@
-FROM fedora:35
+FROM ubuntu:23.04
```

Comment:
would it make sense to use a LTS version here instead?

Reply:
We tried both 22.04 (LTS) and 23.04 (current). Unfortunately, 22.04 still has gcc 11, but we would prefer to use at least gcc 12 due to the improvements in static analysis. Once static analysis is more stable in the future, we will probably continue tracking LTS.